.github/workflows/release.yml

```diff
@@ -0,0 +1,40 @@
+---
+name: Release
+
+on:
+  workflow_run:
+    types:
+      - completed
+    workflows:
+      - 'ci'
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.workflow_run.conclusion == 'success' && contains(github.event.workflow_run.head_branch, 'release/') }}
+    steps:
+      - name: Get tag
+        run: |
+          branch=${{github.event.workflow_run.head_branch}}
+          echo '::set-output name=tag::'${branch#release/}
+        id: get-tag-step
+
+      - name: "Checkout"
+        uses: actions/checkout@v2
+
+      - name: "Set up Docker Buildx"
+        uses: docker/setup-buildx-action@v1
+
+      - name: "Login to GitHub Container Registry"
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: "Build and push"
+        uses: docker/build-push-action@v2
+        with:
+          file: Dockerfile
+          context: .
+          push: true
+          tags: "ghcr.io/${{ github.repository_owner }}/release/${{ github.event.repository.name }}:${{ steps.get-tag-step.outputs.tag }}"
```

Comment:
@edwarnicke Could you check this repo path?

For cmd-nsc it will be:

`ghcr.io/networkservicemesh/release/cmd-nsc:v1.0.0`

Reply:
Done